java/src/main/java/org/whispersystems/signalservice/api/SignalServiceMessageReceiver.java

```diff
@@ -61,6 +64,17 @@ public SignalServiceMessageReceiver(SignalServiceConfiguration urls,
     this(urls, new StaticCredentialsProvider(user, password, signalingKey), userAgent, listener);
   }
 
+  public void setSignalThread(org.whispersystems.signalservice.api.util.SignalThread signalThread) {
```

Comment:
Is there a reason this class is fully qualified here, but also imported? Also is there a reason there needs to be a setter?